test/models/report_test.rb

```diff
@@ -3,7 +3,24 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#editable? returns true when created by yourself' do
```

Comment:
by yourself に違和感がある。`editable?(target_user)` に合わせて by the target user とした方がより自然かも？

Reply:
メソッド名に引数も含めた方が文がわかりやすくなりそう 
```suggestion
  test '#editable?(target_user) returns true when created by the target user' do
```